### PR TITLE
Removed all occurences of geneva

### PR DESCRIPTION
**Pull request by emielsteerneman**
Removed any and all occurences of the geneva drive. All targets are still compiling succesfully.

### CODE REVIEW

File: test/ControlTests/ShotControllerTest.cpp [hunk 1]
```diff
@@ -180,82 +149,13 @@ TEST(ShotControllerTest, getshotdata_test) {
...
-        rtt::world::world->updateWorld(world);
-        auto robot = world::world->getUs().at(0);
-        robot->setGenevaState(3);
-        robot->setWorkingGeneva(true);
-        robot->setTimeToChangeOneGenevaState(-1);
-
-        shotdata = shotController.getRobotCommand(*robot, testhelpers::WorldHelper::getRandomFieldPosition(field), true, BallSpeed::MAX_SPEED, true, ShotPrecision::HIGH);
-        EXPECT_FALSE(shotdata.kicker);
-        EXPECT_FLOAT_EQ(shotdata.geneva, 3);
-    }
```
Comment (rolfvdhulst) [inline test/ControlTests/ShotControllerTest.cpp:209]: This tests more than just the geneva. Have you tested if ShotController still works without the geneva? I don't like these tests being removed.

Comment (rolfvdhulst) [reply test/ControlTests/ShotControllerTest.cpp:209]: Same goes for all other tests in this file.

**Review by rolfvdhulst — CHANGES_REQUESTED** [review]
Are there really no changes in ShotController at all? Seems weird to me, since you edited the test so much, and since ShotController computes geneva state. I heavily suspect you introduced some bugs.

File: test/ControlTests/ShotControllerTest.cpp (hunk 1 above)
Comment (RobotJente) [reply test/ControlTests/ShotControllerTest.cpp:209]: I'm not 100% sure but I don't think the shotcontroller is used anymore. @Ratoone can you confirm this?

Comment (emielsteerneman) [reply test/ControlTests/ShotControllerTest.cpp:209]: The only occurrences of "ShotController" anywhere is in the file ShotControllerTest.cpp, and that file does not belong to any target.

Comment (rolfvdhulst) [reply test/ControlTests/ShotControllerTest.cpp:209]: Ah ok. Then IMO we remove the entire class aswell, as it's functionality relied on geneva a lot on the past.

Comment (emielsteerneman) [reply test/ControlTests/ShotControllerTest.cpp:209]: Can we merge this first?

Comment (rolfvdhulst) [reply test/ControlTests/ShotControllerTest.cpp:209]: Why not remove the class? I'll even do it for you if you don't want to. Keeping code that is unused, has no use and which tests have been removed makes no sense.

Comment (emielsteerneman) [reply test/ControlTests/ShotControllerTest.cpp:209]: If you could do it, that'd be great! I have no idea about the structure of the code anymore

**Review by rolfvdhulst — APPROVED** [review]
LGTM

**Comment by JaroKuiken** [comment]
Never mind, in a different PR. LGTM

**Review by JaroKuiken — APPROVED** [review]

**Pull request merged**